packages/bff/src/routers/authorizationRouter.ts

```diff
@@ -0,0 +1,65 @@
+import { ZodiosEndpointDefinitions } from "@zodios/core";
+import { ZodiosRouter } from "@zodios/express";
+import { bffApi } from "pagopa-interop-api-clients";
+import {
+  ExpressContext,
+  InteropTokenGenerator,
+  SessionTokenGenerator,
+  ZodiosContext,
+  fromAppContext,
+  zodiosValidationErrorToApiProblem,
+} from "pagopa-interop-commons";
+import { makeApiProblem } from "../model/domain/errors.js";
+import { PagoPAInteropBeClients } from "../providers/clientProvider.js";
+import { authorizationServiceBuilder } from "../services/authorizationService.js";
+import { reverseSessionTokenErrorMapper } from "../utilities/errorMappers.js";
+import { config } from "../config/config.js";
+
+const authorizationRouter = (
+  ctx: ZodiosContext,
+  { tenantProcessClient }: PagoPAInteropBeClients,
+  allowList: string[]
+): ZodiosRouter<ZodiosEndpointDefinitions, ExpressContext> => {
+  const authorizationRouter = ctx.router(bffApi.authorizationApi.api, {
+    validationErrorHandler: zodiosValidationErrorToApiProblem,
+  });
+
+  const interopTokenGenerator = new InteropTokenGenerator(config);
+  const sessionTokenGenerator = new SessionTokenGenerator(config);
+  const authorizationService = authorizationServiceBuilder(
+    interopTokenGenerator,
+    sessionTokenGenerator,
+    tenantProcessClient,
+    allowList
+  );
+
+  authorizationRouter
+    .post("/session/tokens", async (req, res) => {
+      const { identity_token: identityToken } = req.body;
+      const { correlationId, logger } = fromAppContext(req.ctx);
+
+      logger.debug("Received request to /session/tokens");
+
+      try {
+        const session_token = await authorizationService.getSessionToken(
+          correlationId,
+          identityToken,
+          logger
+        );
+        return res.status(200).send({ session_token });
+      } catch (error) {
+        const err = makeApiProblem(
+          error,
+          reverseSessionTokenErrorMapper,
```

Comment:
just curious, why `reverse`?

Reply:
I personally don't like it but I found other error mappers called in this way, I'll change it 👍